### PR TITLE
Editing Toolkit: Update to 2.17.0

### PR DESCRIPTION
**Pull request by apeatling**
#### Changes proposed in this Pull Request

* Bumps version of Editing Toolkit to 2.17.0

#### [Changes included in this release](https://github.com/Automattic/wp-calypso/commits/trunk/apps/editing-toolkit)

- [x] WelcomeTour: capture mousedown event and call preventDefault on the event (https://github.com/Automattic/wp-calypso/pull/49583)
- [x] Premium Content: Remove from the Editing Toolkit plugin (https://github.com/Automattic/wp-calypso/pull/49561)
- [x] Anchor: Gutenboarding Launch works immediately without domain or plan selection (https://github.com/Automattic/wp-calypso/pull/48938)
- [x] Editing Toolkit: Update Newspack blocks to v1.19.0 (#49493)
- [x] Plans Store: Always pass locale to plans (#49559)
- [x] Welcome Tour: Prevents clicks on Tour from bubbling up to the Block Inserter (#49583)

#### Testing instructions

1. Load the diff on your sandbox and confirm that the above changes work correctly
2. Test that the new version of the plugin doesn't break Atomic sites by following the instructions in the "Atomic Testing" section at PCYsg-ly5-p2.

### CODE REVIEW

**Comment by apeatling** [comment]
@Automattic/serenity Pinging for a review of the Anchor work in this ETK release. 👍

**Comment by apeatling** [comment]
Adding DO NOT MERGE tag until we have merged the Premium Content block in Jetpack diffs to wpcom.

**Comment by dkoo** [comment]
Can we squeeze https://github.com/Automattic/wp-calypso/pull/49493 into this release?

**Comment by mreishus** [comment]
Re: Anchor: Gutenboarding Launch works immediately without domain or plan selection (#48938)

### Sandbox/WPCOM

* I was able to verify the new functionality when using the new version of FSE.

### Atomic Ephemeral

* I was able to upload the new plugin and verify the site does not break.
* I was not able to verify the new behavior since it only exists on WoA (a "launch" button).

**Review by mreishus — APPROVED** [review]

**Comment by ramonjd** [comment]
> WelcomeTour: capture mousedown event and call preventDefault on the event (#49583)

Tested on simple sites:

Feb-03-2021 20-01-28


And Atomic Ephemeral (even though this feature won't run on Atomic sites)

Feb-03-2021 20-16-35


Welcome tour functions as expected. The click events are captured so that the inserter doesn't close when losing focus. ✅

**Comment by stacimc** [comment]
Rebased to include #49493 -- this also pulled in #49559 and #49583. Pinging @dkoo, @autumnfjeld, and @alshakero

**Review by stacimc — APPROVED** [review]
Tested Premium Content block.

Tested uploading the new plugin on Atomic.

I also tested #49493, #49559, and #49583 following the test instructions from the PRs. Everything looks good to me.

**Review by andrewserong — APPROVED** [review]
Tested that the Premium Content block is loading and working correctly from the Jetpack plugin with this release applied 👍

**Comment by andrewserong** [comment]
D56328-code has been deployed, will wait a little while before then deploying to Atomic.

**Comment by andrewserong** [comment]
And, pushed to .org: https://wordpress.org/plugins/full-site-editing/#developers

**Comment by apeatling** [comment]
Thanks to @stacimc, @andrewserong and those who helped test to get this merged!

**Pull request merged**
